lib/puppet/provider/cisco_bgp/nxapi.rb

```diff
@@ -230,7 +232,11 @@ def puts_config
     current = sprintf("\n%30s: %s", 'asn', @bgp_vrf.asnum)
     current.concat(sprintf("\n%30s: %s", 'vrf', @bgp_vrf.vrf))
     BGP_ALL_PROPS.each do |prop|
-      current.concat(sprintf("\n%30s: %s", prop, @bgp_vrf.send(prop)))
+      begin
+          current.concat(sprintf("\n%30s: %s", prop, @bgp_vrf.send(prop)))
+        rescue Cisco::UnsupportedError
+          nil
+        end
```

Comment:
Indentation is off.  What's strange is that I ran rubocop 0.34.2 against this file, and it's not catching it.